asreview/webapp/src/PreReviewComponents/PreReviewZone.js

```diff
@@ -69,6 +68,11 @@ const PreReviewZone = (props) => {
     ready: false
   });
 
+  const [project, setProject] = React.useState({
+    "id": null,
+    "mode": null
+  })  
+
```

Comment:
I'm a bit concerned about this. I like the idea of a project object. It might be a bit confusing to have this redefined here. It might be more intuitive to extend the Redux store.

Reply:
Removed the idea of a project state 'object'